configuration/configuration.go

```diff
@@ -247,10 +252,13 @@ func (storage Storage) Type() string {
 		case "redirect":
 			// allow configuration of redirect
 		default:
-			return k
+			if storageType != "" {
+				panic("multiple storage drivers specified in configuration")
```

Comment:
this will panic if an environment variable has been set which specifies a storage